docs/source/developer_guides/troubleshooting.md

```diff
@@ -81,3 +81,27 @@ You should probably TRAIN this model on a down-stream task to be able to use it
 ```
 
 The mentioned layers should be added to `modules_to_save` in the config to avoid the described problem.
+
+### Extending the vocabulary
+
+For many language fine-tuning tasks, it's necessary to extend the vocabulary of the model, since new tokens are being introduced. When this is done, the embedding layer needs to be extended to account for the new tokens. But this means that the embedding layer should be stored in addition to the adapter weights when saving the adapter.
```

Comment:
```suggestion
For many language fine-tuning tasks, extending the model's vocabulary is necessary since new tokens are being introduced. This requires extending the embedding layer to account for the new tokens and also storing the embedding layer in addition to the adapter weights when saving the adapter.
```